keps/sig-apps/2255-pod-cost/README.md

```diff
@@ -207,7 +207,7 @@ since users need to explicitly set the annotation.
 None.
 
 * **Were upgrade and rollback tested? Was upgrade->downgrade->upgrade path tested?**
-We will do manual testing.
+Manually tested.
```

Comment:
> Line 196: were those unit tests added? If not, can we make them beta requirement?

Yes, existed since alpha.

> Here - any findings?

Works as expected.